src/sentry/tasks/reports.py

```diff
@@ -898,10 +913,11 @@ def deliver_organization_user_report(timestamp, duration, organization_id, user_
     reports = dict(
         filter(
             lambda item: all(predicate(interval, item) for predicate in inclusion_predicates),
-            zip(projects, backend.fetch(timestamp, duration, organization, projects)),
+            zip(projects, redis_report_backend.fetch(timestamp, duration, organization, projects)),
         )
     )
 
+    set_tag("report.available", not reports)
```

Comment:
I have not been able to determine why I can't get reports generated for me locally even though I have data for the project I belong to. And because of that, I believe we may want to know if there are some production transactions that cannot generate reports.

<img width="977" alt="image" src="https://user-images.githubusercontent.com/44410/189205856-691b70e6-f85a-45bb-9a38-bf7f8aead1ac.png">